stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1311,6 +1323,62 @@
       (ok { stacker: stacker,
             unlock-burn-height: new-unlock-ht }))))
 
+;; This function allows a node to register a new 'current-signer-set' during the prepare phase
+(define-private (update-current-signer-set (new-signer-set (list 4096 {signer: (buff 33)})))
+    (let 
+        (
+            (next-reward-cycle (+ u1 current-pox-reward-cycle))
```

Comment:
Did you test that this code passes type-check?  Because, I'm not seeing the definition of `current-pox-reward-cycle`.